app/models/comment.rb

```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+  validates :post_id, presence: true
+  validates :user_id, presence: true
```

Comment:
- [ ] These validations are unnecessary because you are using belongs_to, let's remove them